### ISSUE TITLE
cannot create regular file '/home/user/TEST_FILE

### ISSUE THREAD

**Issue by sirakai-pc** [a-barinov/liteqube#4]

**Comment by sirakai-pc** [comment]
[liteqube_install.log](https://github.com/a-barinov/liteqube/files/9003673/liteqube_install.log)

could not paste the log file anywhere bcs text is too long.

**Comment by a-barinov** [comment]
That's a pretty stupid bug: installer naively assumes your dom0 username is 'user' which is unlikely to be the case. Do you mind if I fix this in the next commit without doing a 0.92.1 release?

**Comment by sirakai-pc** [comment]
No problem, tell me when u do so

**Comment by a-barinov** [comment]
Yes, [this commit](https://github.com/a-barinov/liteqube/commit/9c1cdbb37ec4105cb4af3f117f16ace476c5d6ce) fixes it.

**Comment by sirakai-pc** [comment]
Got other error in 3.USB:
```
VM core-usb ALREADY EXISTS
CONFIGURING core-usb
STARTING debian-core
CONFIGURING debian-core
Reading package lists.
Building dependency tree...
Reading state information...
Package qubes-usb-proxy is
not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
E: Unable to locate package usbguard
E: Package
"gubes-usb-proxy
has no installation candidate
```

**Comment by Lppsoeht** [comment]
> Got other error in 3.USB:
> ```
> VM core-usb ALREADY EXISTS
> CONFIGURING core-usb
> STARTING debian-core
> CONFIGURING debian-core
> Reading package lists.
> Building dependency tree...
> Reading state information...
> Package qubes-usb-proxy is
> not available, but is referred to by another package.
> This may mean that the package is missing, has been obsoleted, or
> is only available from another source
> E: Unable to locate package usbguard
> E: Package
> "gubes-usb-proxy
> has no installation candidate
> ```
> 

Same error on my machine

**Comment by a-barinov** [comment]
qubes-usb-proxy certainly exists in Qubes debian repo. What this might indicate is package lists were not properly updated over tor, which can easily happen if a bad exit node got picked up.

What I suggest you try doing is open terminal in debian-core (via '~/bin/lq-xtem debian-core') and then run one of 'apt-get update', 'aptitude update' or just aptitude and update package lists from it.

Once update completes properly, qubes-usb-proxy shall get picked up.

**Comment by sirakai-pc** [comment]
Manager to update and install those 2 packages
Tried rerunning install.sh on 2.Network
Got this:
![IMG_20220630_235339_341.jpg](https://user-images.githubusercontent.com/54984910/176785546-312c48ed-0d5b-47a1-bab7-7082c8f118ca.jpg)

**Comment by sirakai-pc** [comment]
Installed drivers for the WiFi usb drive, still stuck at connecting to the internet ![IMG_20220701_011358_972.jpg](https://user-images.githubusercontent.com/54984910/176793622-688e59da-a551-4ed8-8efc-d6d1c69b4798.jpg)

**Comment by a-barinov** [comment]
First, you can automate driver installation by putting driver package name into the installer script, line 11.

Second, the installer definitely falls for you in a non-intuitive way as default permissions for liteqube.signalWiFi should be set by it. Unconditionally.

Any chance for you to run the install script with 'set -x' set? This would give me some clues on what's failing for you.

**Comment by sirakai-pc** [comment]
What do u mean "set -x set"?
Entering the command and retrying the install?
If so there's not much change in the stout

**Comment by sirakai-pc** [comment]
Ohh, and... Not all realtek firmwares are in apt repositories, better installing it manually

**Comment by sirakai-pc** [comment]
Question: can I use sys-usb to attach usb wifi device to net-core?
It just seems the only thing that could work to get internet and continue the installation with net-core and other new qubes.
The installer automatically add realtek usb drive to the selected PCI device for net-core, this causes a bunch of conflict errors from other new qubes.

**Comment by sirakai-pc** [comment]
On the defeat qubes os I had to uninstall wpa_supplicant and use iwd because otherwise it wouldn't work

**Comment by a-barinov** [comment]
> What do u mean "set -x set"? Entering the command and retrying the install? If so there's not much change in the stout

Apologies, I mean adding 'set -x' line as second line of the install script. This will print every command executed which s the most verbose form of debug available.

**Comment by a-barinov** [comment]
> 

> Ohh, and... Not all realtek firmwares are in apt repositories, better installing it manually

The support I have for this is putting your firmware files (not sideloaded packages bu actual firmware files!) into 2.Network/files/Firmware, these will get installed automatically.

**Comment by a-barinov** [comment]
> Question: can I use sys-usb to attach usb wifi device to net-core? It just seems the only thing that could work to get internet and continue the installation with net-core and other new qubes. The installer automatically add realtek usb drive to the selected PCI device for net-core, this causes a bunch of conflict errors from other new qubes.

Two thoughts on this:

1. You can use core-usb to provide network (or core-net to provide usb) but liteqube does not currently support this. You will need to manually merge the relevant folders in /etc/protect and assign devices though.
2. You can use qvm-usb to pass your network dongle to core-net once it boots. Some scripting in dom0 required, but this will work for sure.

BTW, the logic installer uses to assign PCI devided to core-net is super-simple: it copies attached devices list from sys-net. I can probably create a blacklist option if it helps you, but un-assigning the troubled device from sys-net before the install is probably best.
